lib/presentation/widgets/stopover_card.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:flutter/material.dart';
+
+class StopoverCard extends StatefulWidget {
+  final String startStation;
+  final String distance;
+  final String departureArrival;
+  final String track;
```

Comment:
Hierfür die MeansOfTransportEntity verwenden und auf deren Properties zugreifen. (Da kommt noch eine StationEntity als Feld in der MeansOfTransportEntity mit dazu)